spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Restaurant, type: :model do
+  describe 'create' do
+    context 'when succeed' do
+      subject { create(:restaurant) }
+      it 'should create a review metadata item' do
+        expect(subject.restaurant_reviews_metadatum.reviews_count).to eql(0)
+      end
+    end
+  end
+
+  describe 'create' do
+    context 'when no matching cuisine' do
+      it 'should throw an error' do
+        expect { FactoryGirl.create(:restaurant, :without_cuisine) }.to raise_error
```

Comment:
using `create` actually calls the database which is slllooooowwww. try to use `build` whenever possible. checking the model's validity doesn't require persisting it to the db, you can just do 

```
expect { build(:restaurant, cuisine: null) }.not_to be_valid
```

the only exception is if you're planning to actually verify a db-level constraint and then you have to try to persist.